sources/common/toddl_test.go

```diff
@@ -428,3 +428,34 @@ func Test_SchemaToSpannerSequenceHelper(t *testing.T) {
 		assert.Equal(t, expectedConv, conv)
 	}
 }
+
+func Test_cvtCheckContraint(t *testing.T) {
```

Comment:
Please add a mock for "GetSpannerValidExpression" here and check results for when this doesn't return all expressions as valid

Reply:
Removed "GetSpannerValidExpression" from the toddl.go, so no need to add the mock for the API call